<paste log output from Kubernetes-mediated prediction, here>

$ sh ./run_kubernetes.sh
Error from server (AlreadyExists): pods "ml-prediction" already exists
NAME            READY   STATUS    RESTARTS   AGE
ml-prediction   1/1     Running   0          44s
Forwarding from 127.0.0.1:8000 -> 80
Forwarding from [::1]:8000 -> 80
Handling connection for 8000

[2020-07-05 19:52:38,600] INFO in app: JSON payload:
{'CHAS': {'0': 0}, 'RM': {'0': 6.575}, 'TAX': {'0': 296.0}, 'PTRATIO': {'0': 15.3}, 'B': {'0': 396.9}, 'LSTAT': {'0': 4.98}}
[2020-07-05 19:52:38,610] INFO in app: Inference payload DataFrame:
   CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
[2020-07-05 19:52:38,618] INFO in app: Scaling Payload:
   CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
[2020-07-05 19:52:38,621] INFO in app: Prediction value: [20.35373177134412]
127.0.0.1 - - [05/Jul/2020 19:52:38] "POST /predict HTTP/1.1" 200 -